packages/nimble-components/src/utilities/unit-format/tests/decimal-unit-format.spec.ts

```diff
@@ -144,15 +144,15 @@ describe('DecimalUnitFormat', () => {
     });
 
     describe('with unit', () => {
-        class TestScaledUnitFormat extends ScaledUnitFormat {
+        class TestScaledUnitFormat extends IntlNumberFormatScaledUnitFormat {
```

Comment:
Can you explain why this change was made for the test unit scale? Maybe consistency?
Seems pretty much identical to the one in default-unit-format.spec.ts now, should it be moved to a shared fixture?

Reply:
Yeah, it was to be consistent, but I didn't go the next step to share it. I've now moved it into a separate file that they both import from.